components/Storyblok/SbInteriorPage.tsx

```diff
@@ -1,89 +1,180 @@
+      <CreateBloks blokSection={localHeader} slug={slug} />
+      <main id="main-content">
         <article className="bg-fog-light">
-          {props.blok.headerStyle === 'has-image' && (
-            <HeaderWithImage {...props} />
-          )}
-          {props.blok.headerStyle === 'no-image' && (
-            <HeaderNoImage {...props} />
-          )}
-          {props.blok.headerStyle === 'minimal' && <HeaderMinimal {...props} />}
-          {props.blok.headerStyle === 'full-width-image' && (
-            <HeaderFullWidthImage {...props} />
-          )}
-          {props.blok.aboveContent != null &&
-            Object.keys(props.blok.aboveContent).length > 0 && (
-              <div className="ood-interior-page__above-body">
-                <CreateBloks blokSection={props.blok.aboveContent} />
-              </div>
+          {/* Header */}
+          <Container as="header" width="full" mb={headerSpacingBottom} className="break-words">
+            {/* Mobile content (section) menu */}
+            {showMobileContentMenu &&
+              <Container
+                pt={2}
+                pb={hasHeroImage ? 2 : undefined}
+                className={hasHeroImage ? 'bg-palo-alto-dark' : darkBgColors[headerBackgroundColor || 'palo-alto-dark']}
+              >
+                <CreateBloks blokSection={contentMenu} slug={slug} />
+              </Container>
+            }
+            {headerStyle === 'has-image' && (
+              <HeaderSmallImage
+                title={title}
+                intro={intro}
+                headerImage={headerImage}
+                headerBackgroundColor={headerBackgroundColor}
+              />
+            )}
+            {headerStyle === 'no-image' && (
+              <HeaderNoImage
+                title={title}
+                intro={intro}
+                headerBackgroundColor={headerBackgroundColor}
+                hasContentMenu={showMobileContentMenu}
+              />
             )}
-          {(props.blok.bodyTitle ||
-            (props.blok.pageContent != null && Object.keys(props.blok.pageContent).length > 0)) && (
-              <section className="ood-interior-page__body">
-                {props.blok.bodyTitle && (
-                  <header className="centered-container ood-interior-page__body-header text-left">
-                    <Heading className="ood-interior-page__body-header-title ood-has-tab-before">
-                      {props.blok.bodyTitle}
-                    </Heading>
-                  </header>
+            {headerStyle === 'minimal' && <HeaderMinimal title={title} headerBackgroundColor={headerBackgroundColor} />}
+            {headerStyle === 'full-width-image' && (
+              <HeaderFullWidthImage
+                title={title}
+                intro={intro}
+                headerImage={headerImage}
+                headerLogo={headerLogo}
+                visibleVertical={visibleVertical}
+              />
+            )}
+          </Container>
```

Comment:
I'll probably move this into a `Header` component later when I do the final clean up